internal/cmd/gocdk/init.go

```diff
@@ -149,18 +150,26 @@ func inferModulePath(ctx context.Context, pctx *processContext, projectDir strin
 	// so Output can collect stdout.
```

Comment:
Not related to this line but above: let's remove the TODO about #2016 since this PR fixes it.